vignettes/available-sass-variables.Rmd

```diff
@@ -0,0 +1,251 @@
+---
+title: "Available Sass variables"
+output: rmarkdown::html_vignette
+author: "Nick Strayer"
+date: "`r Sys.Date()`"
+vignette: >
+  %\VignetteIndexEntry{available-sass-variables}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  echo = FALSE, 
+  message = FALSE
+)
+
+
+htmltools::tagList(rmarkdown::html_dependency_font_awesome())
+```
+
+```{r setup}
+library(dplyr)
+library(readr)
+library(stringr)
+library(purrr)
+library(glue)
+library(bslib)
+```
```

Comment:
I think we'll probably need to add these to https://github.com/rstudio/bslib/blob/master/.github/workflows/pkgdown.yaml 

(which redeploys the pkgdown site everytime we push to master)

Reply:
I think I did this correctly here: https://github.com/rstudio/bslib/pull/238/commits/14094cee7df7dd3178eeb72f67c7f7dcdc494628